web/src/styles/scss/print.scss

```diff
@@ -30,7 +30,7 @@
   .btn-collapse svg,
   .alert,
   .rdw-editor-toolbar,
-  .table-frozen-left-pane {
+  .ds-c-review .nowrap {
```

Comment:
Neat! I was expecting to put a `visibility--screen` class on the edit bar, but I like this approach better. Would it make sense to make a special class for the edit bar instead of targeting the `nowrap` though?  🤷‍♂

Reply:
Thanks to the weird way I daisy-chained these, it's now tough for me to go back and edit this for this PR. But I'll fix it in the next one!